pkg/apis/openebs.io/zfs/v1/zz_generated.deepcopy.go

```diff
@@ -29,6 +29,7 @@ import (
 func (in *Pool) DeepCopyInto(out *Pool) {
 	*out = *in
 	out.Free = in.Free.DeepCopy()
+	out.Used = in.Used.DeepCopy()
```

Comment:
ouch, could this have caused any bugs?

Reply:
Not really. It'd require another thread to be working on this exact resource at the same time.